runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/BpmnModeler/BpmnModeler.tsx

```diff
@@ -266,6 +257,16 @@ const BpmnModeler = React.forwardRef<ModelerImperativeHandle, ModelerProps>(
             modeler?.get('commandStack')?.redo();
         };
 
+    
+        // great debugging log for auto apply, uncomment if needed
```

Comment:
maybe its better to move it to another file like `BpmnModeler.utils.ts` and just add the comment instead of injecting commented out code in the component

Reply:
sure!